cursive-core/src/printer.rs

```diff
@@ -407,13 +409,14 @@ impl<'a, 'b> Printer<'a, 'b> {
     /// # let b = backend::Dummy::init();
     /// # let t = theme::load_default();
     /// # let printer = Printer::new((6,4), &t, &*b);
-    /// printer.print_box((0, 0), (6, 4), false);
+    /// printer.print_box((0, 0), (6, 4), false, false);
     /// ```
     pub fn print_box<T: Into<Vec2>, S: Into<Vec2>>(
         &self,
         start: T,
         size: S,
         invert: bool,
+        highlight: bool,
```

Comment:
I think at this point it may be better to use some parameter-struct (to have named parameters) rather than additional parameters:

```rust
use BoxStyle as BS;

printer.print_box(start, size, BS { invert: false, highlight: false });
```